### ISSUE TITLE
0.6.0 - ANSI plugin!

### ISSUE THREAD

**Issue by xomx** [nea/MarkdownViewerPlusPlus#15]
both x86 and x64 plugin flavor refused by the UNICODE Notepad++

**Comment by nea** [comment]
Damn it. The direct build runs in my Notepad. Maybe its my new build process.

I will check.


Sorry about that

**Comment by dail8859** [comment]
Just FYI it looks like the new DLL is not exporting any functions at all. This is what `dumpbin` shows for v0.5.1

```
> dumpbin /exports MarkdownViewerPlusPlus.dll
Microsoft (R) COFF/PE Dumper Version 12.00.40629.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Dump of file MarkdownViewerPlusPlus.dll

File Type: DLL

  Section contains the following exports for \MarkdownViewerPlusPlus.dll

    00000000 characteristics
    58A3FDE3 time date stamp Wed Feb 15 02:06:11 2017
        0.00 version
           0 ordinal base
           6 number of functions
           6 number of names

    ordinal hint RVA      name

          5    0 001E5B8E beNotified
          2    1 001E5B5E getFuncsArray
          4    2 001E5B7E getName
          0    3 001E5B3E isUnicode
          3    4 001E5B6E messageProc
          1    5 001E5B4E setInfo

  Summary

        2000 .reloc
        2000 .rsrc
        2000 .sdata
      1E4000 .text

```

---

And this is v0.6.0

```
> dumpbin /exports MarkdownViewerPlusPlus.dll
Microsoft (R) COFF/PE Dumper Version 12.00.40629.0
Copyright (C) Microsoft Corporation.  All rights reserved.


Dump of file MarkdownViewerPlusPlus.dll

File Type: DLL

  Summary

        2000 .reloc
        2000 .rsrc
      1F6000 .text

```

**Comment by nea** [comment]
Heyho

I feel so stupid! When I created my automated release zip build I forgot to DllExport, and therefore: nothing to call for Notepad++ ^^' And I tested with an old version on my own Notepad++ as the copying task was overridden by me, too.

I corrected that issue and updated the release zips in 0.6.0 https://github.com/nea/MarkdownViewerPlusPlus/releases/tag/0.6.0

I tried the ZIP DLL's in my own x86 and x64 instances now, and it worked.


I am really sorry about that inconvenience!
